src/app/pages/login-page/login-page.component.ts

```diff
@@ -11,8 +13,10 @@ import { NgIf } from '@angular/common';
   styleUrl: './login-page.component.scss',
 })
 export class LoginPageComponent {
+  constructor(private authService: AuthService) {}
+  public router = inject(Router);
+  public errorMessage = '';
   public isPasswordShown = false;
```

Comment:
better to use signals
```suggestion
  public errorMessage = signal('');
  public isPasswordShown = signal(false);
```